configure_bazel.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 Google LLC
```

Comment:
Can this go in build_tools?

Reply:
yeah build_tools/bazel/ seems nice

Reply:
I feel moderately strongly (and there is plenty of precedent) that levers needed to meaningfully do anything with the project belong in the root. Having a configure script in the root is a pretty strong signal that some action is expected to be taken prior to just jumping in.